.travis.yml

```diff
@@ -1,29 +1,24 @@
 matrix:
   include:
-  - name: "DApp unit tests"
+  - name: "origin-contracts / origin-js / origin-dapp unit tests"
```

Comment:
By combining all these test targets, we lose the benefits of running tests in parallel.
I understand the install will be done once for all these targets and that's a gain.
But the tests themselves may take some time to run and by losinig parallelization we may overall be worse off in terms of overall time it takes to run the Travis CI build ?

Travis CI gives a break down of how much time is spent running each step (e.g. npm install vs npm test) so we should be able to get data to decide...